python_modules/dagster/dagster_tests/storage_tests/test_io_manager.py

```diff
@@ -1086,3 +1086,34 @@ def another_asset(an_asset: int) -> int:
         defs.load_asset_value("another_asset", instance=instance)
 
         assert executed["yes"]
```

Comment:
It would be neat if we could write a test to gather publicly exported resources from our packages and check that they all have the flag set

Reply:
yes! i'm working on this in another branch